src/openfermion/utils/_remove_symmetry_qubits.py

```diff
@@ -0,0 +1,118 @@
+""" Module to remove two qubits from the problem space
+    using conservation of electron number and conservation
+    of electron spin. As described in arXiv:1701.08213.
+"""
+
+
+import openfermion
+from openfermion.ops import FermionOperator, QubitOperator
+from openfermion.transforms import get_fermion_operator, bravyi_kitaev_tree
+from openfermion.utils import reorder, up_then_down, prune_unused_indices
+import numpy
+
+
+
+
+
+
+
+def remove_symmetry_qubits(fermion_hamil, active_orbitals, active_electrons):
+    """ Returns the qubit Hamiltonian for the molecular fermionic
+        Hamiltonian supplied, with two qubits removed using conservation
+        of electron spin and number, as described in arXiv:1701.08213.
+
+        Args:
+            fermion_hamil: A fermionic molecular hamiltonian obtained
+                           using OpenFermion. An instance of the
+                           FermionOperator class.
+
+            active_orbitals: Int type object. The number of active orbitals
+                             being considered for the molecule.
+
+            active_electrons: Int type object. The number of active electrons
+                              being considered for the molecule (note, this
+                              is less than the number of electrons in the
+                              molecule if some orbitals have been assumed
+                              filled).
+
+        Returns:
+                qubit_hamil: The qubit Hamiltonian corresponding to
+                             the supplied fermionic Hamiltonian, with
+                             two qubits removed using spin symmetries.
+
+        Notes: This function reorders the spin orbitals as all spin-up, then
```

Comment:
The convention in OpenFermion is to keep the even-odd convention in spin orderings. So I object to this returning all then down by default. There is a function called "reorder" in utils/_operator_utils.py which assists with reorderings.